financial-templates-lib/clients/ExpiringMultiPartyClient.js

```diff
@@ -58,7 +58,7 @@ class ExpiringMultiPartyClient {
   // Whether the given undisputed `liquidation` (`getUndisputedLiquidations` returns an array of `liquidation`s) is disputable.
   // `tokenRedemptionValue` should be the redemption value at `liquidation.time`.
   isDisputable = (liquidation, tokenRedemptionValue) => {
-    return !this._isUnderCollateralized(liquidation.numTokens, liquidation.amountCollateral, tokenRedemptionValue);
+    return !this._isUnderCollateralized(liquidation.numTokens, liquidation.liquidatedCollateral, tokenRedemptionValue);
```

Comment:
+1 This is a really good catch. If we had had any partial liquidations this would have caused an issue.

Reply:
It actually was correctly using `amountCollateral === liquidatedCollateral` but its safer to be explicit